spec/commands/csv_data_upload_spec.rb

```diff
@@ -49,14 +49,53 @@
       legislations_csv,
       new_records: 2, not_changed_records: 0, rows: 2, updated_records: 0
     )
+
+    legislation = Legislation.find_by(title: 'Climate Law')
+
+    expect(legislation.legislation_type).to eq('legislative')
+    expect(legislation.description).to eq('Description')
+    expect(legislation.document_types_list).to include('Law')
+    expect(legislation.visibility_status).to eq('pending')
+    expect(legislation.geography.iso).to eq('POL')
+    expect(legislation.frameworks.size).to eq(2)
+    expect(legislation.frameworks_list).to include('Mitigation', 'Adaptation')
+    expect(legislation.keywords.size).to eq(2)
+    expect(legislation.keywords_list).to include('keyword1', 'keyword3')
+    expect(legislation.natural_hazards.size).to eq(2)
+    expect(legislation.natural_hazards_list).to include('sharkinados', 'flooding')
   end
 
   it 'imports CSV files with Litigation data' do
+    legislation1 = create(:legislation)
+    legislation2 = create(:legislation)
+    updated_litigation = create(:litigation)
+
+    csv_content = <<-CSV
+      Id,Title,Document type,Geography iso,Jurisdiction iso,Citation reference number,Summary,Keywords,Core objective,Visibility status,Legislation ids
+      ,Litigation number 1,Case,GBR,GBR,EWHC 2752,Lyle requested judicial review,"keyword1,keyword2",Objectives,pending,"#{legislation1.id}, #{legislation2.id}"
+      #{updated_litigation.id},Litigation number 2,Case,GBR,GBR,[2013] NIQB 24,The applicants were brothers ...,,,Draft,
+    CSV
+
+    litigations_csv = fixture_file('litigations.csv', content: csv_content)
+
     expect_data_upload_results(
       Litigation,
       litigations_csv,
-      new_records: 4, not_changed_records: 0, rows: 4, updated_records: 0
+      new_records: 1, not_changed_records: 0, rows: 2, updated_records: 1
     )
+
+    litigation = Litigation.find_by(title: 'Litigation number 1')
+
+    expect(litigation.jurisdiction.iso).to eq('GBR')
```

Comment:
same here: `have_attributes()` matcher could be more readable.

Reply:
hmm, have_attributes nice idea :+1: